interface/modules/zend_modules/module/Carecoordination/config/module.config.php

```diff
@@ -155,7 +155,7 @@
                 return new CarecoordinationTable();
             },
             EncounterccdadispatchTable::class =>  function (ContainerInterface $container, $requestedName) {
-                    return new EncounterccdadispatchTable($container->get(\Laminas\Db\Adapter\Adapter::class));
```

Comment:
@sjpadgett ,
Just seeing what happens with phpstan here.
To confirm, the above line is needed for ccda to work, correct:
```
return new EncounterccdadispatchTable($container->get(\Laminas\Db\Adapter\Adapter::class));
```

Reply:
kind of a head scratcher, huh!